packages/codemods/src/codeMods/utilities/props.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  JsxAttribute,
+  JsxOpeningElement,
+  JsxSelfClosingElement,
+  SyntaxKind,
+  VariableDeclarationKind,
+  ts,
+  Node,
+} from 'ts-morph';
+import { EnumMap } from '../types';
+
+export function renameProp(
+  instances: (JsxOpeningElement | JsxSelfClosingElement)[],
+  toRename: string,
+  replacementName: string,
+  changeValueMap?: EnumMap<string>,
+  replacementValue?: string,
+) {
+  instances.forEach(val => {
+    /* For each instance, first see if desired prop exists in the open. */
+    let foundProp = val.getAttribute(toRename);
+    if (foundProp) {
+      /* If found, do a simple name-replacementName. */
+      foundProp.set({ name: replacementName });
+      if (changeValueMap) {
+        /* Change the value of the enum via map conversion. */
+        let enumExp = (foundProp as JsxAttribute)
+          .getFirstChildByKind(SyntaxKind.JsxExpression)
```

Comment:
I think that there need to be some more null/undefined checks. The getFirstChildByKind can return undefined I believe.

Reply:
Good point! I'll fix this first thing tomorrow!